routes/users.js

```diff
@@ -135,7 +129,7 @@ usersRouter.patch('/:id', async (req, res, next)=>{
 usersRouter.delete("/:id", isAdmin, async (req, res, next) => {
     try {
       const { id } = req.params;
-      const deletedUser = await deleteUser(id);
+      const deletedUser = await deleteUser(req.id);
```

Comment:
curious why we changed this to `req.id` if we pull `id` out of `req.params` on the line above?